projects/cm_mcu/commands/SensorControl.c

```diff
@@ -533,6 +541,20 @@ BaseType_t alarm_ctl(int argc, char **argv, char *m)
       return pdFALSE;
     }
   }
+  else if (strcmp(argv[1], "setvolt") == 0) {
+    snprintf(m, s, "alarm voltages are fixed to ADC thresholds\r\n");
+    return pdFALSE;
+  }
```

Comment:
what's the point of this command? it seems to not do anything?

Reply:
Sorry, this should have been removed. Done.